sim/midas/src/main/scala/midas/passes/partition/Utils.scala

```diff
@@ -0,0 +1,58 @@
+package midas.passes.partition
+
+import scala.annotation.tailrec
+
+import firrtl._
+import firrtl.ir._
+import firrtl.Mappers._
+import firrtl.annotations._
+import firrtl.passes.{ResolveKinds, InferTypes}
```

Comment:
Nit: Across scala files I see inconsistent spacing, extra imports. Can you clean that up a bit?